src/compiler/output-targets/dist-custom-elements/index.ts

```diff
@@ -66,7 +66,7 @@ export const outputCustomElements = async (
  * @returns bundle options suitable for generating a rollup configuration
  */
 export const getBundleOptions = (
-  config: d.Config,
+  config: d.ValidatedConfig,
```

Comment:
Similar to my comment above, can we update the JSDoc `@param` description for `config` here as well?
```diff
- * @param config user-supplied Stencil configuration
+ * @param config the validated compiler configuration we're using
```